src/VsCodeDialog/vscode.ts

```diff
@@ -119,23 +144,22 @@ export const listInstalledExtensions = async (): Promise<VsCodeExtension[]> => {
     }));
 };
 
-export const getNrfjprogStatus = () => {
-    return new Promise<boolean>(resolve => {
-        const spawnAsync = spawn('nrfjprog', {
-            shell: true,
-            env: {
-                ...remote.process.env,
-                PATH: pathEnvVariable(),
-            },
-        });
-
-        spawnAsync.on('close', (code, signal) => {
-            if (code === 0 && signal === null) {
-                return resolve(true);
+export const getNrfjprogStatus = async () => {
+    try {
+        await spawnAsync('nrfjprog');
+        try {
+            if (isAppleSilicon) {
+                const stdout = await spawnAsync('file $(which jlinkexe)');
```

Comment:
Have you tested this on windows? I don't think `file` and `which` is part of the standard commands. There is the `where`-command that does something similar, but I think the $() won't work.

Reply:
This isn't being run on windows